src/sql/base/browser/ui/table/plugins/buttonColumn.plugin.ts

```diff
@@ -102,7 +108,7 @@ export class ButtonColumn<T extends Slick.SlickData> implements Slick.Plugin<T>
 	}
 
 	private isCurrentColumn(columnIndex: number): boolean {
-		return this._grid.getColumns()[columnIndex].id === this.definition.id;
+		return this._grid?.getColumns()[columnIndex]?.id === this.definition.id;
```

Comment:
why do we need to add the ?

Reply:
when I was debugging, I noticed at some point this method was called when grid didn't have any columns. checking for null or adding ?fixed that

---
In reply to: [457877015](https://github.com/microsoft/azuredatastudio/pull/10918#discussion_r457877015) [](ancestors = 457877015)